data/json/mapgen/map_extras/roadblock_mil.json

```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "update_mapgen_id": "mx_roadblock_mil",
+    "object": {
+      "rows": [
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----bbbbbbb--bbbbbbb--  ",
+        "----------------------  ",
+        "----------------------  ",
+        "------------t---------  ",
+        "------------ssss------  ",
+        "-------------Lg-------  ",
+        "------------ssss------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "----------------------  ",
+        "-----------t----------  ",
+        "----------------------  ",
+        "----bbbbbbb--bbbbbbb--  ",
+        "                        ",
+        "                        "
+      ],
+      "furniture": { "s": "f_sandbag_half", "g": "f_generator_broken", "b": "f_barricade_road" },
```

Comment:
If the generator is broken, how do turrets and searchlights still work?
I thought ASRG was added as an explanation to that.